File: app/src/main/java/com/boardgamegeek/util/fabric/PlayManipulationEvent.java
package com.boardgamegeek.util.fabric;

import com.crashlytics.android.answers.Answers;

public class PlayManipulationEvent extends DataManipulationEvent {
	public static void log(String action, String gameName) {
		PlayManipulationEvent event = new PlayManipulationEvent(action);
		event.putCustomAttribute("GameName", gameName);
		Answers.getInstance().logCustom(event);
	}

	protected PlayManipulationEvent(String action) {
		super("Play");
		putCustomAttribute("action", action);
	}
}

File: app/src/main/java/com/boardgamegeek/util/fabric/PlayManipulationEvent.kt
package com.boardgamegeek.util.fabric

import com.crashlytics.android.answers.Answers

class PlayManipulationEvent(action: String) : DataManipulationEvent("Play") {

    init {
        putCustomAttribute("action", action)
    }

    companion object {
        @JvmStatic
        fun log(action: String, gameName: String) {
            val event = PlayManipulationEvent(action).apply {
                putCustomAttribute("GameName", gameName)
            }
            Answers.getInstance().logCustom(event)
        }
    }
}
